src/main/java/uk/gov/hmcts/juror/api/moj/xerox/letters/SummonsLetter.java

```diff
@@ -29,6 +30,16 @@ public SummonsLetter(JurorPool jurorPool,
             .build());
     }
 
+    @Override
+    protected void setup(Juror juror) {
```

Comment:
do we need Juror parameter as its not used?

Reply:
We have to provide it to override the base setup method unfortunately.